apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1086,11 +1086,14 @@ export class CaseService {
       })
     }
 
+    // TODO: Use subpoenas already included in theCase.defendants
+    // - no need to call the subpoena service
+    // - then add to transition tests
     const subpoenasToRevoke = await this.subpoenaService.findByCaseId(
       theCase.id,
     )
 
-    if (subpoenasToRevoke?.length > 0) {
+    if (theCase.origin === CaseOrigin.LOKE && subpoenasToRevoke?.length > 0) {
```

Comment:
The case must be created from LÖKE, otherwise the upload fails. This prevents us spamming LÖKE with unsuccessful calls.